code/modules/projectiles/projectile/magic.dm

```diff
@@ -283,6 +287,9 @@
 
 	to_chat(new_mob, "<span class='warning'>Your form morphs into that of a [randomize].</span>")
 
+	if((istype(new_mob, /mob/living/silicon/robot/modules/syndicate) || istype(new_mob, /mob/living/carbon/alien/humanoid) || istype(new_mob, /mob/living/simple_animal/hostile)))
+		to_chat(new_mob, "<span class='userdanger'>Despite taking the form of an antagonistic being, you have the same mind as before your transformation. Your loyalties and interests remain the same. Unless you were turned into a shade, or were previously an antagonist, this is not a pass to go antagonize the station.</span>")
+
 	var/poly_msg = CONFIG_GET(keyed_list/policy)["polymorph"]
 	if(poly_msg)
 		to_chat(new_mob, poly_msg)
```

Comment:
Wouldn't the message for polymorph come under this message?

Reply:
(Also if you get turned into a xeno, I'm pretty sure you get an antagonist role and an objective)

Reply:
You do get that as a xeno yet will get banned if you act on it

Reply:
We really ought to seperate xenobio transformations from wizard transformations